app/src/main/java/com/example/nvdovin/weatherapp/domain/utils/time/TimeUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.example.nvdovin.weatherapp.domain.utils.time;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+public class TimeUtils {
+
+    public static final int MILLISECONDS = 1000;
+    private static final int ONE_DAY = 1;
+    private static final int RESET_VALUE = 0;
+    private static final int PERIOD = 3;
+    public static final int COUNT_OF_RECORDINGS_FOR_DAY = 8;
+
+    public Long[] getAllPeriodsForDay(Long timestamp) {
+        Long dates[] = new Long[COUNT_OF_RECORDINGS_FOR_DAY];
+        Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
```

Comment:
"UTC" may also be retrieved in a constant a reused along the entire class, something like "UTC_TIMEZONE_ID"